cmake/Options.cmake

```
@@ -23,3 +23,8 @@ option(TP_BUILD_TESTING "Build tests" OFF)
 
 # Force to build libuv from the included submodule
 option(TP_BUILD_LIBUV "Build libuv from source" OFF)
+
+# Directories
+include(GNUInstallDirs)
+set(TP_INSTALL_LIBDIR ${CMAKE_INSTALL_LIBDIR} CACHE STRING "Directory in which to install libraries")
```

To make this essentially a no-op shouldn't it be set by default to `${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}`
```suggestion
set(TP_INSTALL_LIBDIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR} CACHE STRING "Directory in which to install libraries")
```